### PR TITLE
Show how long each request took

### PR DESCRIPTION
**Pull request by DFoxinator**

### CODE REVIEW

File: src/js/XHRLogger.js [hunk 1]
```diff
@@ -111,6 +111,29 @@ XHRLogger.prototype.logToConsole = function(content, encoding) {
 };
 
 /**
+ * Format the request timings
+ */
+XHRLogger.prototype.formatRequestTime = function(timings) {
+	var total_ms = 0,
+		stage;
+		
+	delete timings.ssl;
```
Comment (weikinhuang) [inline src/js/XHRLogger.js]: try not to modify the objects inline.
you can do Object.keys(timings).forEach(); and then if you encounter ssl skip it. Or we can add a extend or copy function

**Comment by weikinhuang** [comment]
otherwise looking good

**Comment by DFoxinator** [comment]
Thanks, I moved the ssl check into the -1 check so it no longer modified the object.

File: src/js/XHRLogger.js [hunk 2]
```diff
@@ -111,6 +111,27 @@ XHRLogger.prototype.logToConsole = function(content, encoding) {
...
 
 /**
+ * Format the request timings
+ */
+XHRLogger.prototype.formatRequestTime = function(timings) {
+	var total_ms = 0,
+		stage;
+	
+	for (stage in timings) {
+		if (timings[stage] === -1 || stage === "ssl") {
```
Comment (weikinhuang) [inline src/js/XHRLogger.js]: also consider `hasOwnProperty` check

**Comment by weikinhuang** [comment]
+1 otherwise

**Pull request merged**